src/NovelBasicFillRect.cpp

```diff
@@ -74,4 +58,26 @@ RGBAConfig NovelBasicFillRect::getColourConfig() const {
 void NovelBasicFillRect::setColourConfig(const RGBAConfig& value) {
   _colourConfig = value;
 }
+void NovelBasicFillRect::configureObjectBuffers(bool refreshBuffers) {
+  NovelRenderObject::configureObjectBuffers(refreshBuffers);
+
+  if(refreshBuffers) {
+    auto config = getColourConfig();
+    _colourData = {
+        config.getRScalar(), config.getGScalar(), config.getBScalar(), config.getAScalar(),

```

Comment:
I am unsure. I'd assume "yes" but I can rework it if you like.

Reply:
@tannergooding is there a concern about how expensive this would be?
I think with its current state this would be okay. However, if we can check to see if the compiler optimizes this, I think we should do so before we resolve it.

Reply:
I've already reworked it in commit 3d3a404.